### PR TITLE
Fix issue #8.

### PR DESCRIPTION
**Pull request by thebestmensch**
Add multiparty dm cleaner capabilities.
Update readme to reflect new functionality.

### CODE REVIEW

File: slack_cleaner/args.py [hunk 1]
```diff
@@ -34,7 +34,8 @@ def __init__(self):
                             help='Direct message\'s name, e.g., sherry')
         g_chan.add_argument('--group',
                             help='Private group\'s name')
-
+        g_chan.add_argument('--mpdirect',
+                            help='Multiparty direct message\'s name, e.g., sherry,james')
```
Comment (kfei) [inline slack_cleaner/args.py:38]: Looks pretty good to me. Concise and handy. :+1:

File: slack_cleaner/cli.py [hunk 2]
```diff
@@ -187,6 +189,18 @@ def get_direct_id_by_name(name):
                 return i['id']
 
 
+def get_mpdirect_id_by_name(name):
+    res = slack.mpim.list().body
+    name = 'mpdm-' + name + '-1'
```
Comment (kfei) [inline slack_cleaner/cli.py]: So it makes `--mpdirect sherry,james` different from `--mpdirect james,sherry`.  Another approach I can think of is finding the group id by matching `set(names.split(","))` and `set(res['groups']['members'])`.  Also, the naming policy `'mpdm-' + names + '-1'` seems more likely to change over time than `res['groups']['members']`.

**Comment by kfei** [comment]
@JMensch Many thanks for this PR! :smile: Please have a look at my in-line comments and let me know your thoughts.

File: slack_cleaner/cli.py (hunk 2 above)
Comment (thebestmensch) [inline slack_cleaner/cli.py]: That sounds great! I'll update the pull request. Thanks!

**Comment by thebestmensch** [comment]
@kfei check out my changes when you get a chance!

**Comment by kfei** [comment]
@JMensch Nice! Also, thank you for the bonus typo and trailing-spaces fix. :smile:

**Pull request merged**